### PR TITLE
Added SOLID and initial donation

### PR DESCRIPTION
**Pull request by jbradach**

### CODE REVIEW

**Comment by kiriakos** [comment]
Hi there! I'm trying to understand the intermediate transaction that You posted. This seems to be a transaction of 100Ada to a wallet staked to SOLID pool. How does this transaction fund a donation? Also, the PDF mentions an amount of ` $100 ADA` Is the donation in Dolars or in Ada?

**Comment by kiriakos** [comment]
PS, who is the owner of this wallet? They have 9M unstaked and I would like to propose the delegate it towards MDP :-)

**Comment by jbradach** [comment]
> Hi there! I'm trying to understand the intermediate transaction that You posted (tx cd6e6b1d94cd506e8ffb67c7b5ef440d318b6ed2316914131724e1d0193ddf34). This seems to be a transaction of 100Ada to a wallet staked to SOLID pool. How does this transaction fund a donation? Also, the PDF mentions an amount of ` $100 ADA` Is the donation in Dolars or in Ada?

Hello! Sorry for the delayed response. This donation of 100 ADA was made to a wallet controlled by the organization. They're new to cryptocurrency and this is the first contribution of this type they've received. The PDF mentioning dollars is a mistake since they received 100 ADA and at the time of the donation that was around $160.

The organization opted to keep the contribution as ADA and stake it back into the pool until they are prepared to convert it to USD at a later time.

**Comment by jbradach** [comment]
> PS, who is the owner of this wallet? They have 9M unstaked and I would like to propose the delegate it towards MDP :-)

I'm not sure where you're seeing 9M unstaked. I don't have anything close to that in my wallets. One of wallet does have 10,000,000 SOLID, a governance token we created recently that we plan to use to allow delegators to vote on which projects we fund.

File: SOLID/donations.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    contact: uvidaho@gmail.com
+
+  ada: 100 # <- These must be numbers. Can be decimals though. See below
+  usd: 163 # Only usd currency accepted, check on xe.com for exchange rate.
+  timestamp: 1620431928 
+  # ^ This is a unix timestamp of the moment you donated.
+  #   You can go to https://www.unixtimestamp.com/ to calculate your donation.
+
+  files:
+  # ^ a list of files relating to the donation. Usualy photos or screenshots
```
Comment (kiriakos) [inline SOLID/donations.yml:16]: Hmm... apparently the `yq` utility, which is being used by the CI to validate the file references, has a bug and returns the comments as well. Can you please remove the comments in lines 16-19?
Thanks

**Review by kiriakos — CHANGES_REQUESTED** [review]

**Review by jbradach — COMMENTED** [review]
Comments have been removed.

**Review by kiriakos — APPROVED** [review]

**Review by kiriakos — APPROVED** [review]
lgtm

**Comment by ResponsibleStaking** [comment]
Cant merge because of "This branch is out-of-date with the base branch".
Instructions here should help to update the branch to allow merging this PR: https://stackoverflow.com/questions/53712930/how-to-fix-this-in-git-this-branch-is-out-of-date-with-the-base-branch

**Comment by kiriakos** [comment]
@jbradach I had the same issue with a couple other PRs and here is a writeup of what I did that allowed the PRs to be merged. #129 Can we try that?

**Pull request closed without merging**